llpc/test/shaderdb/general/TestCombineOfMultipleStoreInstructions.frag

```diff
@@ -0,0 +1,87 @@
+#version 460
```

Comment:
Can you add a test that shows how the IR changes?

Reply:
I will update the test when Nicolai will merge his PR #2740